### PR TITLE
[TF 2.0 API docs] expanding the tf.keras.activations.relu

### PR DESCRIPTION
**Pull request by Williscool13**
Addressing issue #26211. Feedback would be greatly appreciated, and will make any revisions necessary.

### CODE REVIEW

**Review by mihaimaruseac — DISMISSED** [review]

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by mihaimaruseac** [comment]
```
FAIL: Found 1 non-whitelisted pylint errors:
tensorflow/python/keras/activations.py:220: [C0301(line-too-long), ] Line too long (82/80)
```

Please fix.

**Review by mihaimaruseac — APPROVED** [review]

**Review by rthadur — DISMISSED** [review]

**Comment by rthadur** [comment]
@Williscool13 can you please check this failures here https://source.cloud.google.com/results/invocations/d5139b8a-2820-46ed-8563-f604013d9fda/log

**Comment by mihaimaruseac** [comment]
```
File "/b/f/w/bazel-out/k8-opt/bin/tensorflow/tools/docs/tf_doctest.runfiles/org_tensorflow/tensorflow/python/keras/activations.py", line 204, in tensorflow.python.keras.activations.relu
Failed example:
    tf.keras.activaions.relu(foo, max_value=5).numpy()
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python3.6/doctest.py", line 1330, in __run
        compileflags, 1), test.globs)
      File "<doctest tensorflow.python.keras.activations.relu[3]>", line 1, in <module>
        tf.keras.activaions.relu(foo, max_value=5).numpy()
    AttributeError: module 'tensorflow.python.keras.api._v2.keras' has no attribute 'activaions'
```

please fix

File: tensorflow/python/keras/activations.py [hunk 1]
```diff
@@ -186,23 +186,40 @@ def softsign(x):
...
+  change the max value of the activation,
+  and to use a non-zero multiple of the input for values below the threshold.
+
+  For example:
+  >>> foo = tf.constant([-10, -5, 0.0, 5, 10], dtype = tf.float32)
+  >>> tf.keras.activations.relu(foo).numpy()
+  # ==> array([0., 0., 0., 5., 10.], dtype=float32)
+  >>> tf.keras.activations.relu(foo, alpha=0.5).numpy()
+  # ==> array([-5. , -2.5,  0. ,  5. , 10. ], dtype=float32)
+  >>> tf.keras.activaions.relu(foo, max_value=5).numpy()
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/activations.py]: Typo, should be `activations`

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**